### DISCUSSION TITLE
Why do I load the third font pTexture is nullptr

### DISCUSSION THREAD

**Discussion by shimenga** [ocornut/imgui#5956] [New Users Build/Link/Run issues ONLY!]
io.Fonts->AddFontFromFileTTF(R"(C:\Windows\Fonts\msyh.ttc)", 18.0f, nullptr, io.Fonts->GetGlyphRangesChineseFull());
    Logofont = io.Fonts->AddFontFromMemoryTTF((void*)DiPaiTi_data, DiPaiTi_size, 71.0f, nullptr, io.Fonts->GetGlyphRangesChineseFull());
    Modulefont = io.Fonts->AddFontFromMemoryTTF((void*)YeZiGongChangRunYuanTi_data,YeZiGongChangRunYuanTi_size, 25, nullptr, io.Fonts->GetGlyphRangesChineseFull());

**Comment by ocornut** [comment]
71.0f size + oversample + GetGlyphRangesChineseFull() is too big and won't fit.

From:
https://github.com/ocornut/imgui/blob/master/docs/FONTS.md#font-loading-instructions

"If you have very large number of glyphs or multiple fonts, the texture may become too big for your graphics API. The typical result of failing to upload a texture is if every glyph appears as a white rectangle."
